recipes/qxmpp/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(QXmpp REQUIRED)
```

```suggestion
find_package(QXmpp REQUIRED CONFIG)
```

The `cmake_find_package_multi` generator creates Cmake config files only, not Findxxx.cmake